### ISSUE TITLE
HostnameVerifier

### ISSUE THREAD

**Issue by bitsofinfo** [OrbitzWorldwide/consul-client#145]
In a prior version we used to be able to do this, but its gone. How can I do this in 0.12.4?

```
consulBuilder.withClientBuilder(ClientBuilder.newBuilder().hostnameVerifier(new HostnameVerifier() {
					            public boolean verify(String s, SSLSession sslSession) {
					                return true;
					            }
					        }));
```

**Comment by rickfast** [comment]
The client was changed to use a shaded version of retrofit and OkHTTP. I think we can expose it in the Consil builder. I will try and get to it soon.

**Comment by bitsofinfo** [comment]
Or will it pick it up from this one? http://download.java.net/jdk7/archive/b123/docs/api/javax/net/ssl/HttpsURLConnection.html

Just need to get this working to upgrade this lib we wrote

**Comment by rickfast** [comment]
fixed by b6c64d78cd96c4833e49e82febc290f3e76c22f3. release in `0.12.5`

**Comment by bitsofinfo** [comment]
thanks!
